pom.xml

```diff
@@ -224,6 +224,11 @@
                 <artifactId>nimbus-jose-jwt</artifactId>
                 <version>${nimbusds.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Is this dependency available in the runtime?

Reply:
dependecy is removed in 789a83b